community/dejavu-ttf/build

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh -e
+
+install -d "$1/usr/share/fonts/TTF"
```

Comment:
Remove this line and change the next line to `-Dm644`.

Reply:
Copy pasta